File: Java/LinguagemC/1160 - Crescimento Populacional.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
    int t, pa, pb, tempo; 
    double ga, gb, pat, pbt;
    cin >> t;
    for(int i = 1; i <= t; i++){
        tempo = 0;
        cin >> pa >> pb >> ga >> gb;
        while (pa <= pb) {
            pa *= (ga/100.0)+1.0;
            pb *= (gb/100.0)+1.0;
            tempo++;
            if(tempo > 100){
                cout << "Mais de 1 seculo." << endl;
                break;
            }
        }
        if(tempo<=100){
            cout << tempo << " anos." << endl;
        }
    }
    return 0;
}

File: Java/Python/1160-CrescimentoPopulacional.py
t = int(input())
for i in range(0,t):
    tempo = 0
    pa, pb, ga, gb=map(float, input().split(" "))
    pa = int(pa)
    pb = int(pb)
    while pa<=pb:
        pa = int(pa*(1 + ga/100))
        pb = int(pb*(1 + gb/100))
        tempo = tempo + 1
        if tempo >= 101:
            print("Mais de 1 seculo.")
            break
    if tempo <= 100:
        tempo = int(tempo)
        print(f'{tempo} anos.')